CONTRIBUTING.md

```diff
@@ -35,6 +35,11 @@ You will also need to install the Docker engine:
 - [Docker for Mac](https://docs.docker.com/engine/installation/mac/)
 - [Docker for Windows](https://docs.docker.com/engine/installation/windows/)
 - [Docker for Linux](https://docs.docker.com/engine/installation/linux/ubuntulinux/)
+ 
+Install [gox](https://github.com/mitchellh/gox) (v1.14+). For Mac and Linux:
+  ```bash
+  brew install gox
+  ```
```

Comment:
I had to have it installed for `make control-plane-dev`. Since that was mentioned a few steps later I thought it meant it was required. If you still think it should be removed let me know and I'll take it out.

Reply:
Ah! ok to keep it then!